src/JoomlaBrowser.php

```diff
@@ -133,12 +133,11 @@ public function installJoomla()
         $I->debug('I select dk-DK as installation language');
         // Select a random language to force reloading of the lang strings after selecting English
         $I->selectOptionInChosen('#jform_language', 'Danish (DK)');
-        $I->waitForText('Generel konfiguration', 10, 'h3');
+        $I->waitForText('Generel konfiguration', 20, 'h3');
```

Comment:
good catch! :+1: 

Please turn it to 60 instead of 20 as suggested at https://docs.joomla.org/Testing_Joomla_Extensions_with_Codeception#How_much_time_should_I_wait_for_an_Element_or_Text.3F